### PR TITLE
Job posting and deleting

### PR DESCRIPTION
**Pull request by donatgosalcii**

### CODE REVIEW

File: ReviveIT/Application/Interfaces/IJobPostFeature.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using Application.DTO;
+
+namespace Application.Interfaces
+{
+    public interface IJobPostFeature
+    {
+        Task<JobPostResultDto> CreateJobPostAsync(JobPostDto jobPostDto, string userId);
+        Task<DeleteJobDto> DeleteJobPostAsync(int jobId, string userId);
+
```
Comment (EdisonAlushaj) [inline ReviveIT/Application/Interfaces/IJobPostFeature.cs:9]: Delete the white line in line 9

File: ReviveIT/WebUI/program.cs [hunk 2]
```diff
@@ -83,12 +83,23 @@
...
 
+builder.Services.AddControllers()
+    .AddJsonOptions(options =>
+    {
+        options.JsonSerializerOptions.ReferenceHandler = System.Text.Json.Serialization.ReferenceHandler.IgnoreCycles;
+        options.JsonSerializerOptions.DefaultIgnoreCondition = System.Text.Json.Serialization.JsonIgnoreCondition.WhenWritingNull;
+        options.JsonSerializerOptions.WriteIndented = true;
+    });
+
+
```
Comment (EdisonAlushaj) [inline ReviveIT/WebUI/program.cs:101]: Delete the white line'Sin row 101 and 102

**Review by EdisonAlushaj — CHANGES_REQUESTED** [review]

File: ReviveIT/Application/Interfaces/IJobPostFeature.cs (hunk 1 above)
Comment (donatgosalcii) [reply ReviveIT/Application/Interfaces/IJobPostFeature.cs]: Done

File: ReviveIT/WebUI/program.cs (hunk 2 above)
Comment (donatgosalcii) [reply ReviveIT/WebUI/program.cs]: Done

File: ReviveIT/Application/DTO/DeleteJobDto.cs [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        }
+
+        public static DeleteJobDto Success(string message, object data = null, int statusCode = 200) =>
+            new DeleteJobDto(true, message, data, statusCode);
+
+        public static DeleteJobDto Failure(string message, int statusCode = 400) =>
+            new DeleteJobDto(false, message, null, statusCode);
+    }
+}
+
```
Comment (EdisonAlushaj) [inline ReviveIT/Application/DTO/DeleteJobDto.cs:25]: Delete the white line

File: ReviveIT/Application/Features/User/JobPostFeature.cs [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+            }
+
+            _context.Jobs.Remove(jobPost);
+            await _context.SaveChangesAsync();
+
+            return DeleteJobDto.Success("Job post deleted successfully.");
+        }
+    }
+}
+
```
Comment (EdisonAlushaj) [inline ReviveIT/Application/Features/User/JobPostFeature.cs:88]: Delete the white line

**Review by EdisonAlushaj — CHANGES_REQUESTED** [review]

**Pull request merged**